lib/src/main/java/io/ably/lib/realtime/AblyRealtime.java

```diff
@@ -71,6 +71,13 @@ public void onAuthUpdated() {
 	}
 
 	/**
+	 * Authentication error occurred
+	 */
+	public void onAuthError(ErrorInfo errorInfo) {
```

Comment:
In fact, whatever calls `onAuthError` could instead just call `connection.connectionManager.onAuthError` surely which will at least reduce the exposure of methods / objects that users should not be using.